Source/WebCore/xml/XMLViewer.css

```diff
@@ -85,11 +85,11 @@ div.collapsible > div.hidden {
 }
 
 .collapse-button {
-    background-image: -webkit-canvas(arrowDown);
+    background: url("data:image/svg+xml,<svg xmlns='http://www.w3.org/2000/svg' fill='%23909090' width='10' height='10'><path d='M0 0 L8 0 L4 7 Z'/></svg>");
     height: 10px;
 }
 
 .expand-button {
-    background-image: -webkit-canvas(arrowRight);
-    height: 11px;
+    background: url("data:image/svg+xml,<svg xmlns='http://www.w3.org/2000/svg' fill='%23909090' width='10' height='10'><path d='M0 0 L0 8 L7 4 Z'/></svg>");
```

Comment:
Can we set the fill color to `#5a5a5a` to match the old color?

Reply:
Sure - once EWS run finishes, before merging, I will do it.

Reply:
<img width="525" alt="image" src="https://github.com/user-attachments/assets/cc6e2879-6ba7-4a67-b533-0518a37c2ff5" />

^ As suggested.

Reply:
Nice, I think the contrast looks much better

Reply:
Pushed. :-)